mkdir -p `dirname results/LFV_808v1/AnalyzeLFVMuTaufakesdown/data_SingleMuon_Run2016B_PromptReco-v2_25ns.root`
(in /afs/hep.wisc.edu/cms/fmeng/Fanbo/CMSSW_8_0_8/src/UWHiggs/lfvmutau)
warning: Insecure world writable dir /afs/hep.wisc.edu/cms/fmeng in PATH, mode 040777
time mega AnalyzeLFVMuTaufakesdown.py inputs/LFV_808v1/data_SingleMuon_Run2016B_PromptReco-v2_25ns.txt results/LFV_808v1/AnalyzeLFVMuTaufakesdown/data_SingleMuon_Run2016B_PromptReco-v2_25ns.root --workers 2 --chain 10
#using farmout:0
#dry run:0
RuntimeWarning:class RooDataHistEffBuilder already in TClassTable
RuntimeWarning:class RooCruijff already in TClassTable
Error in <TUnixSystem::FindDynamicLibrary>: $CMSSW_BASE/lib/$SCRAM_ARCH/libHiggsAnalysisCombinedLimit.so does not exist in /afs/hep.wisc.edu/cms/fmeng/Fanbo/CMSSW_8_0_8/biglib/slc6_amd64_gcc530:/afs/hep.wisc.edu/cms/fmeng/Fanbo/CMSSW_8_0_8/lib/slc6_amd64_gcc530:/afs/hep.wisc.edu/cms/fmeng/Fanbo/CMSSW_8_0_8/external/slc6_amd64_gcc530/lib:/cvmfs/cms.cern.ch/slc6_amd64_gcc530/cms/cmssw/CMSSW_8_0_8/biglib/slc6_amd64_gcc530:/cvmfs/cms.cern.ch/slc6_amd64_gcc530/cms/cmssw/CMSSW_8_0_8/lib/slc6_amd64_gcc530:/cvmfs/cms.cern.ch/slc6_amd64_gcc530/cms/cmssw/CMSSW_8_0_8/external/slc6_amd64_gcc530/lib:/cvmfs/cms.cern.ch/slc6_amd64_gcc530/external/llvm/3.7.1-giojec/lib64:/cvmfs/cms.cern.ch/slc6_amd64_gcc530/external/gcc/5.3.0/lib64:/cvmfs/cms.cern.ch/slc6_amd64_gcc530/external/gcc/5.3.0/lib:.:/cvmfs/cms.cern.ch/slc6_amd64_gcc530/lcg/root/6.06.00-ikhhed3/lib:/lib64/tls/x86_64:/lib64/tls:/lib64/x86_64:/lib64:/usr/lib64/tls/x86_64:/usr/lib64/tls:/usr/lib64/x86_64:/usr/lib64
['tPt30', 'tPt40', 'tPt45', 'tPt50', 'mPt25', 'mPt35', 'mPt55', 'mPt30', 'mPt40', 'mPt50', 'deltaPhi2.00', 'deltaPhi3.00', 'deltaPhi2.50', 'deltaPhi2.40', 'deltaPhi2.20', 'tMtToPfMet_type180', 'tMtToPfMet_type135', 'tMtToPfMet_type170', 'tMtToPfMet_type140', 'tMtToPfMet_type150', 'tMtToPfMet_type160']
Running optimization: True
Traceback (most recent call last):
  File "/afs/hep.wisc.edu/cms/fmeng/Fanbo/CMSSW_8_0_8/src/FinalStateAnalysis//PlotTools/scripts/mega", line 94, in <module>
    module = __import__(class_name, fromlist=[class_name])
  File "/cvmfs/cms.cern.ch/slc6_amd64_gcc530/lcg/root/6.06.00-ikhhed3/lib/ROOT.py", line 303, in _importhook
    return _orig_ihook( name, *args, **kwds )
  File "/afs/hep.wisc.edu/cms/fmeng/Fanbo/CMSSW_8_0_8/src/UWHiggs/lfvmutau/AnalyzeLFVMuTaufakesdown.py", line 282, in <module>
    pu_corrector = PileupWeight.PileupWeight('MC_Spring16', *pu_distributions)
  File "/afs/hep.wisc.edu/cms/fmeng/Fanbo/CMSSW_8_0_8/python/FinalStateAnalysis/TagAndProbe/PileupWeight.py", line 55, in __init__
    mc_file =root_open(_MC_PU_DISTRIBUTIONS[mctag])
NameError: global name 'root_open' is not defined
4.02user 1.86system 0:21.32elapsed 27%CPU (0avgtext+0avgdata 185596maxresident)k
575656inputs+40outputs (2435major+71435minor)pagefaults 0swaps
rake aborted!
Command failed with status (1): [time mega AnalyzeLFVMuTaufakesdown.py inpu...]

(See full trace by running task with --trace)
